src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -647,12 +647,18 @@ private FlowPathPayload buildFlowPathPayload(List<FlowPathDto> paths, String flo
     @Override
     public CompletableFuture<List<FlowEventPayload>> listFlowEvents(String flowId,
                                                                     long timestampFrom,
-                                                                    long timestampTo) {
+                                                                    long timestampTo, int maxCount) {
+        if (maxCount < 1) {
+            throw new MessageException(RequestCorrelationId.getId(), System.currentTimeMillis(),
+                    ErrorType.PARAMETERS_INVALID, format("Invalid `max_count` argument '%s'.", maxCount),
+                    "`max_count` argument must be positive.'");
```

Comment:
excess single quote at the end of the sentence -.-

Reply:
will fix